docs/source/installation.rst

```diff
@@ -158,4 +158,19 @@ error log (in Apache on Debian/Ubuntu, for example, this is ``/var/log/apache2/e
 Deployment on Docker
 --------------------
 
-**TODO**
+It is also possible to deploy the server using Docker.
+
+- First build the Docker Image.  From the server/scripts folder run::
```

Comment:
This should be 'From the ``deploy`` directory run:'

Also, can you change the `::` pattern to use the code block syntax like the rest of the examples above please?